lib/scrabble.rb

```diff
@@ -1,18 +1,57 @@
+require './word_finder'
+require './word_list'
+
 class Scrabble
+  attr_accessor :point_values
+
+  def initialize(word, score)
+    @word = word    
+    @score = 0
+    @point_values = ''
+    @letter = ''
+  end
 
   def score(word)
-    1
+   letter = word.upcase.split('')
```

Comment:
I would save this to letters since it is now an array of letters instead. Letter suggests to me it is a single character.